silk/middleware.py

```diff
@@ -64,6 +71,9 @@ def process_request(self, request):
 
 class SilkyMiddleware:
     def __init__(self, get_response):
+        if config.SILKY_AUTHENTICATION and not (set(AUTH_AND_SESSION_MIDDLEWARES) & set(settings.MIDDLEWARE)):
+            raise SilkNotConfigured("SILKY_AUTHENTICATION can not be enabled without Session, Authentication or Message Django's middlewares")
+
         self.get_response = get_response
```

Comment:
Not sure if it's the best place to do the check, but when line `self.get_response = get_response` runs the exception mentioned in the ticked is raised